langlib/lang.string/src/main/ballerina/src/lang.string/string.bal

```diff
@@ -154,3 +154,11 @@ public function toCodePointInts(string str) returns int[] = external;
 # + return - a string with a character for each code point in `codePoints`; or an error
 # if any member of `codePoints` is not a valid code point
 public function fromCodePointInts(int[] codePoints) returns string|error = external;
+
+# Compare two given strings ignoring the case. In the comparison, any non ascii character
```

Comment:
```suggestion
# Compare two given strings ignoring the case. In the comparison, any non ASCII character
```

Reply:
@dulvinw, seems like this documentation is different from what is in the spec. Shall we change to use the same?

https://github.com/ballerina-platform/ballerina-spec/blob/master/lang/lib/string.bal#L127

Reply:
Fixed